packages/db/src/lib/schema.prisma

```diff
@@ -38,3 +39,13 @@ model FarcasterUser {
   user        User     @relation(fields: [userId], references: [id])
   username    String   @unique
 }
+
+model AuthSession {
+  id              String   @id @default(uuid())
+  createdAt       DateTime @default(now())
+  expiresAt       DateTime
+  authToken       String   @unique
+  secondAuthToken String   @unique
```

Comment:
We don't need this. You can just use a library like bcrypt to encrypt the userId and then decrypt the encrypted token you receive on OAuth callback. Will make life easier